src/pages/2022-09-02-davis-brin-dane-evans/index.md

```diff
@@ -0,0 +1,215 @@
+
+![Davis Full Season EPA](/blog_images/davis-brin/brin_full_season_epa.png)
+
+<br />
+
+If we split the season up it becomes a little more clear how Brin improved, while also removing some of that boom and bust style. Here's his EPA from the first six games:
+
+![Davis First Half EPA](/blog_images/davis-brin/brin_first_season_epa.png)
+
+<br />
+
+In those six games he had two (UC Davis and Houston) that with EPA worse than **-15**, meaning he cost us points due to turnovers and/or just not playing very well. And on the flip side, his best game of the season came against Ohio State and he had an EPA just under **15**. Unfortunately that performance wasn't enough since we still ended up losing that one. And as the trendline (the red dashed line) indicates, he was improving over those six games.
+
+Turning our attention to the final seven games, we can see that the lows aren't quite as low and there is more consistency betwen games.
+
+![Davis Second Half EPA](/blog_images/davis-brin/brin_second_season_epa.png)
+
+<br />
+
+His performances with EPAs under zero -Navy, Cincy and Tulane- combined were barely worse than just the Houston game in terms of total EPA. So his bad games were not as bad–-and we won one of those games with a chance to play spoiler at Cincinatti. Additionally, the trend on the back half shows a much steeper improvement from game seven to thirteen. This gives me confidence that Davis Brin had a very normal 'first-year starter' season and makes me excited to see his progression in year two.
+
+I mean, the last time Tulsa had a first-year starter throw 16+ interceptions, with just over 3,000 yards of offense - it worked out pretty well. 
+
+<br />
+
+## Dane Evans' First Season
+
+<br />
+
+Of course, I'm talking about Dane Evans. TU quarterback from 2013-2016 who set AAC records and led Tulsa to a 10-3 season in his senior season. Dane Evans is a name that carries a lot of weight around Tulsa because of the success that he and the team had during his tenure. But, what you may not recall right off the bat is that his first full season as a starter wasn't anything special. Let's look at his 2014 stat line.
+
+<br />
+
+<center>
+
+| **Completions** | **Attempts** | **Percentage** | **Yards** | **Yards / Attempt** | **Touchdowns** | **Interceptions** |
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| 256 | 462 | 55.4 | 3,102 | 6.7 | 23 | 17 |
+
+</center>
+
+<br />
+
+The only stat Evans lead Brin in was touchdowns with 23 to 18. Now, there is the caveat that 2014 was the final year with Bill Blankenship as head coach and he ran a different style compared to what Montgomery would bring the next season so its not exactly an apples to apples comparison. But that's why I spent so much time explaining EPA--so that I can make some sort of comparison.
```

Comment:
The paragraph says Evans had 23 TDs vs 18 INTs, but the table says 23 TDs vs 17 INTs. Which is it?

Reply:
The 18 in that sentence is referring to Brin's TDs not Dane's INTs

Reply:
Ah yeah whoops, that makes sense. Also I think "lead" should be "led" in this same paragraph.

Reply:
Probably, I kept going back and forth in my head lol